Code/Bench/stereo.cpp

```diff
@@ -7,27 +7,63 @@
 #include <GraphMol/Chirality.h>
 #include <GraphMol/ROMol.h>
 #include <GraphMol/SmilesParse/SmilesParse.h>
+#include <GraphMol/MolOps.h>
 
 using namespace RDKit;
 
 TEST_CASE("Chirality::findPotentialStereo", "[stereo]") {
-  for (auto smiles : bench_common::CASES) {
-    auto mol = v2::SmilesParse::MolFromSmiles(smiles);
-    REQUIRE(mol);
+  auto samples = bench_common::load_samples();
 
-    BENCHMARK("Chirality::findPotentialStereo: " + std::string(smiles)) {
-      return Chirality::findPotentialStereo(*mol);
-    };
-  }
+  BENCHMARK("Chirality::findPotentialStereo") {
+    auto total = 0;
+
+    for (auto &mol : samples) {
+      auto stereo_infos = Chirality::findPotentialStereo(mol);
+      mol.clearComputedProps();
+      for (auto &info : stereo_infos) {
+        total += info.controllingAtoms.size();
+      }
+    }
+
+    return total;
+  };
+}
+
+TEST_CASE("CIPLabeler::assignCIPLabels", "[stereo]") {
+  auto samples = bench_common::load_samples();
+  BENCHMARK("CIPLabeler::assignCIPLabels") {
+    auto total = 0;
+    for (auto &mol : samples) {
+      CIPLabeler::assignCIPLabels(mol);
+    }
+    return total;
```

Comment:
`total` is never mutated so optimizer will const fold it and return 0. Keeping total doesn't help in this case (my mistake).

```suggestion
    for (auto &mol : samples) {
      CIPLabeler::assignCIPLabels(mol);
    }
```